test/parallel/test-child-process-sync-process-send.js

```diff
@@ -0,0 +1,23 @@
+var common = require('../common');
+var assert = require('assert');
+var fork = require('child_process').fork;
+var N = 4 << 20;  // 4 MB
+
+for (var big = '*'; big.length < N; big += big);
+
+if (process.argv[2] === 'child') {
+  process.send(big);
+  process.exit(42);
+}
+
+var proc = fork(__filename, ['child']);
+
+proc.on('message', common.mustCall(function(msg) {
+  assert.equal(typeof(msg), 'string');
```

Comment:
is there a reason `typeof(msg)` is used here instead of `typeof msg`? I usually see the latter in the codebase